js/components/operatorSignIn/operatorSignInModal.tsx

```diff
@@ -13,7 +15,20 @@ export const OperatorSignInModal = (): ReactElement => {
         setShow(false);
       }}
     >
-      <OperatorSelection nfcSupported={true} />
+      {badge === null ?
+        <OperatorSelection
+          nfcSupported={true}
+          onOK={(badge: string) => {
+            setBadge(badge);
+          }}
```

Comment:
**suggestion:** I think this would work as well:
```suggestion
          onOK={setBadge}
```

Reply:
fixed! (I forget which commit)